src/Config/Factory.php

```diff
@@ -98,4 +100,54 @@ protected function configureHook(Config\Hook $config, array $json) : void
             $config->addAction(new Config\Action($actionJson['action'], $options, $conditions));
         }
     }
+
+    /**
+     * Append all included configuration to the current configuration
+     *
+     * @param \CaptainHook\App\Config $config
+     * @param array                   $json
+     */
+    private function appendIncludedConfigurations(Config $config, array $json)
+    {
+        $includes = $this->loadIncludedConfigs($json, $config->getPath());
+        foreach (HookUtil::getValidHooks() as $hook => $class) {
+            foreach ($includes as $includedConfig) {
+                $this->copyActionsFromTo($includedConfig->getHookConfig($hook), $config->getHookConfig($hook));
+            }
+        }
+    }
+
+    /**
+     * Read included configurations to add them to the main configuration later
+     *
+     * @param  array  $json
+     * @param  string $path
+     * @return \CaptainHook\App\Config[]
+     */
+    protected function loadIncludedConfigs(array $json, string $path) : array
+    {
+        $includes  = [];
+        $directory = dirname($path);
+        $files     = isset($json['config']['includes']) && is_array($json['config']['includes'])
+                   ? $json['config']['includes']
+                   : [];
+
+        foreach ($files as $file) {
+            $includes[] = self::create($directory . DIRECTORY_SEPARATOR . $file);
+        }
+        return $includes;
+    }
+
+    /**
+     * Append actions for a given hook from all included configurations to the current configuration
```

Comment:
I think that "from a single included configuration" rather than "from all included configurations" would be a more accurate comment here.